src/components/drop-down/drop-down.tsx

```diff
@@ -96,19 +94,19 @@ export class DropDown extends React.PureComponent<DropDownProps, DropDownState>
 
     private toggleDropdown = () => {
         if (!this.props.disabled) {
-            this.setState({open: !this.state.open});
+            this.props.onOpenStateChange && this.props.onOpenStateChange({value: !this.props.open});
```

Comment:
Please set default value for `onOpenStateChange` to noop and you won't have to check if it is truthy.

Reply:
i had a default for this. will return it.
@JoMarton for further references since we talk a lot about this :)